themes/catppuccin-frappe-blue.json

```diff
@@ -4,7 +4,7 @@
         "background": "#303446",
         "block": "#a6d189",
         "cursor": "#f2d5cf",
-        "emphasis": "#e78284",
```

Comment:
All frappé themes were using `red` for emphasis, unlike `maroon` which the rest of the themes were using.